submission/submission-security/src/main/kotlin/ebi/ac/uk/security/util/SecurityUtil.kt

```diff
@@ -17,10 +18,6 @@ import java.time.Clock
 import java.time.OffsetDateTime
 import java.util.UUID
 
-internal const val DEV_KEY = "975dd2ca-58eb-407b-ba0f-858f15f7304d"
-internal const val BETA_KEY = "9c584ae3-678a-4462-b685-54c37a1bc047"
-internal const val PROD_KEY = "01ecc118-dbec-4df8-8fe8-f5cd7364b2b7"
-
 internal const val DEV_INSTANCE = "http://ves-hx-f2.ebi.ac.uk:8120"
```

Comment:
We should move this to the config (in this ticket or a separate one). Will need that for migration to k8s.